guide/oauth2/README.md

```diff
@@ -0,0 +1,200 @@
+# OAuth2 
+
+OAuth2 enables application developers to build applications that utilize authentication and data from the Discord API.
+
+## A quick example
+
+###  Setting up a basic web server
+
+Typically, OAuth2 is used in websites to get information about their users from an external service. Here, we will use Node.js' built-in `http` module to create a web server because our example is simple enough. First, we create a file named `index.js` which will be used to start the server.
+
+```js
+const http = require('http');
+const fs = require('fs');
+const port = 53134;
+
+http.createServer((req, res) => {
+	let responseCode = 404;
+	let content = '404 Error';
+	if (req.url === '/') {
+		responseCode = 200;
+		content = fs.readFileSync('./index.html');
+	}
+	res.writeHead(responseCode, {
```

Comment:
This piece of code + some other areas should be spaced out a bit more, for consistency/readability reasons. Here's a good example of what I mean: https://github.com/discordjs/guide/blob/master/guide/assets/js/dark-theme.js#L13-L31

In this case, just add some blank lines before/after the if statement and after the `.writeHead()` call.